src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.tsx

```diff
@@ -738,10 +738,10 @@ export class EditDataSourceForm extends React.Component<EditDataSourceProps, Edi
             defaultMessage: 'Credential',
           })}
         >
-          <EuiRadioGroup
+          <EuiSelect
             options={credentialSourceOptions}
-            idSelected={this.state.auth.type}
-            onChange={(id) => this.onChangeAuthType(id)}
+            value={this.state.auth.type}
+            onChange={(e) => this.onChangeAuthType(e)}
```

Comment:
```suggestion
            onChange={this.onChangeAuthType}
```

nit: not blocking the PR for this though

Reply:
made the change